### ISSUE TITLE
[Web] Web is off

### ISSUE THREAD

**Issue by AichiChikuwa** [naeruru/mimiuchi#23]
No response from the web.
![image](https://github.com/naeruru/mimiuchi/assets/65875548/1747c79d-6c5d-4e99-bb42-a55775b41316)

**Comment by AichiChikuwa** [comment]
Sorry the screenshot is misleading. It's probably not the actual issue as these logs doesn't exist if I launch it in incognito mode.

**Comment by naeruru** [comment]
what exactly is happening? it looks like one of your extensions may be causing those errors. which would explain why it doesn't happen in incognito mode. 
I recently updated the app so make sure you have the correct version. 0.3.0. It shows on the bottom of the setting nav

**Comment by AichiChikuwa** [comment]
After using it for a while, the STT functionality completely failed. The record switch couldn't be turned on and every time I did so an error log with an empty string displayed (can't recall exactly what, but should be a string returned by console.log()). I tried several times using incognito mode, vpn, and different devices, but none of them worked. 
Again, sorry for the misleading screenshot. That means nothing to the actual issue.
The web works for me now so everything should be all set at this point. I let you know if this issue occurs again. I'd probably record a video at that point to show exactly what's going on. Thanks for your precious time!
